app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryCommentsFragment.kt

```diff
@@ -0,0 +1,439 @@
+                        setOnClickListener {
+                            val span = currentSpan.apply { clearCurrentSpan() }
+                            if (span is URLSpan) {
+                                val activity = context.findActivity<MainActivity>()
+                                if (!activity.jumpToReaderByPage(span.url, galleryDetail)) {
+                                    if (!navigator.navWithUrl(span.url)) {
+                                        activity.openBrowser(span.url)
+                                    }
+                                }
+                            } else {
+                                coroutineScope.launch {
+                                    context.doCommentAction(item)
+                                }
+                            }
+                        }
+                    }
+                }
+                if (comments.hasMore) {
+                    item {
+                        AnimatedVisibility(refreshing) {
+                            Box(modifier = Modifier.fillMaxWidth()) {
+                                CircularProgressIndicator(
+                                    modifier = Modifier.align(Alignment.Center).padding(keylineMargin),
+                                )
+                            }
+                        }
+                        AnimatedVisibility(!refreshing) {
+                            TextButton(
+                                onClick = {
+                                    coroutineScope.launchIO {
+                                        refreshing = true
+                                        runSuspendCatching {
+                                            refreshComment(true)
+                                        }
+                                        refreshing = false
+                                    }
+                                },
+                                modifier = Modifier.fillMaxWidth().padding(keylineMargin),
+                            ) {
+                                Text(text = stringResource(id = R.string.click_more_comments))
+                            }
+                        }
+                    }
+                }
+            }
+            if (commenting) {
+                Surface(
+                    modifier = Modifier.align(Alignment.BottomCenter).onGloballyPositioned { coordinates ->
+                        editTextMeasured = with(density) { coordinates.size.height.toDp() }
+                    },
+                    color = MaterialTheme.colorScheme.primaryContainer,
+                ) {
+                    Row(modifier = Modifier.fillMaxWidth().navigationBarsPadding()) {
+                        BasicTextField2(
+                            value = userComment,
+                            onValueChange = { userComment = it },
+                            modifier = Modifier.weight(1f).padding(keylineMargin),
```

Comment:
The text size is too small compared with the send button

Reply:
Done